games/xworld3d/tasks/xworld3d_task.py

```diff
@@ -153,6 +154,9 @@ def _record_answer(self, answer):
     def _record_target(self, target):
         """
         Record the target for later evaluation and rewarding
+        The target should be a pair tuple, where the first element is
+        a string indicating which task's target is recorded, and the
+        second element is anything to be recorded
```

Comment:
is this requirement satisfied? I see at some places where _record_target are called, the targets does not satisfy this requirement